plone/app/locales/locales/it/LC_MESSAGES/plone.po

```diff
@@ -4394,15 +4394,15 @@ msgstr "Non mostrare il bordo del portlet"
 
 #: CMFPlone/controlpanel/browser/overview.pt:145
 msgid "On"
-msgstr ""
+msgstr "Acceso"
 
 #: plone.app.users/plone/app/users/browser/schemaeditor.py:129
 msgid "One image field maximum."
 msgstr "Massimo un campo immagine"
 
 #: plone.schemaeditor/plone/schemaeditor/browser/schema/delete_fieldset.py:22
 msgid "Only empty fieldsets can be deleted"
-msgstr ""
+msgstr "Solo i fieldset vuoti possono essere cancellati"
```

Comment:
Gia' forse fieldset e' intraducibile...

Reply:
Sì, è un tecnicismo ed è difficile da tradurre. La cosa più carina che mi viene in mente è: "Solo i gruppi di campi vuoti possono essere cancellati" ma non mi convince moltissimo.

Reply:
Anche io rimarrei con "fieldset" perchè "gruppi di campi" non mi suona benissimo.